### PR TITLE
Don't use PolymerGestures

### PR DESCRIPTION
**Pull request by jonnor**
Refs #222 

Problems/TODO:

* [x] onTrackEnd on nodes not always called, causing move transaction to stay open.
* [x] Fix dragging nodes moves them too fast
* [x] Add back support for non-Chrome mobile
* [x] Remove uses of PolymerGestures `up` and `down` events in menus

### CODE REVIEW

**Comment by bergie** [comment]
Can we remove `bower.json` after this PR?

**Comment by jonnor** [comment]
No, to remove Polymer completely also need to rewrite the tests and examples to use the React components instead of Polymer elements.

**Comment by bergie** [comment]
We should also remove the `bower install` postinstall step soonish

**Comment by bergie** [comment]
Some issues related to `panend` upstream: hammerjs/hammer.js#845 hammerjs/hammer.js#707

**Comment by jonnor** [comment]
Removed WIP marker now. Note that there are two minor UX regressions (see CHANGES), but hopefully  these can be fixed later.

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## 0.10.0 (unreleased)
 
+UI changes
+
+* Edges cannot be dropped on target port. Have to tap to complete edge connection.
```
Comment (bergie) [inline CHANGES.md]: That's a pretty nasty regression

Comment (jonnor) [inline CHANGES.md]: Feel free to fix. I don't think I can be arsed to do it now.

**Pull request merged**